2016-02-09 14:59:09 -0500 SEVERE: unable to initialize application
org.jruby.exceptions.RaiseException: (LoadError) no such file to load -- /Users/Fj00/Ts/config/environment
	at org.jruby.RubyKernel.require(org/jruby/RubyKernel.java:1065)
	at RUBY.require(/Users/Fj00/.rvm/rubies/jruby-1.7.16/lib/ruby/shared/rubygems/core_ext/kernel_require.rb:55)
	at RUBY.(root)(/Users/Fj00/Ts/config.ru:3)
	at org.jruby.RubyBasicObject.instance_eval(org/jruby/RubyBasicObject.java:1533)
	at RUBY.initialize(/Users/Fj00/.rvm/gems/jruby-1.7.16/gems/rack-1.6.4/lib/rack/builder.rb:55)
	at RUBY.(root)(/Users/Fj00/Ts/config.ru:1)

2016-02-09 14:59:09 -0500 SEVERE: Exception sending context initialized event to listener instance of class org.jruby.rack.RackServletContextListener
org.jruby.rack.RackInitializationException: no such file to load -- /Users/Fj00/Ts/config/environment
	from org/jruby/RubyKernel.java:1065:in `require'
	from /Users/Fj00/.rvm/rubies/jruby-1.7.16/lib/ruby/shared/rubygems/core_ext/kernel_require.rb:55:in `require'
	from /Users/Fj00/Ts/config.ru:3:in `(root)'
	from org/jruby/RubyBasicObject.java:1533:in `instance_eval'
	from /Users/Fj00/.rvm/gems/jruby-1.7.16/gems/rack-1.6.4/lib/rack/builder.rb:55:in `initialize'
	from /Users/Fj00/Ts/config.ru:1:in `(root)'

	at org.jruby.rack.RackInitializationException.wrap(RackInitializationException.java:29)
	at org.jruby.rack.RackApplicationFactoryDecorator.init(RackApplicationFactoryDecorator.java:104)
	at org.jruby.rack.RackServletContextListener.contextInitialized(RackServletContextListener.java:50)
	at org.apache.catalina.core.StandardContext.listenerStart(StandardContext.java:4994)
	at org.apache.catalina.core.StandardContext.startInternal(StandardContext.java:5492)
	at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:150)
	at org.apache.catalina.core.ContainerBase$StartChild.call(ContainerBase.java:1575)
	at org.apache.catalina.core.ContainerBase$StartChild.call(ContainerBase.java:1565)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: org.jruby.exceptions.RaiseException: (LoadError) no such file to load -- /Users/Fj00/Ts/config/environment
	at org.jruby.RubyKernel.require(org/jruby/RubyKernel.java:1065)
	at RUBY.require(/Users/Fj00/.rvm/rubies/jruby-1.7.16/lib/ruby/shared/rubygems/core_ext/kernel_require.rb:55)
	at RUBY.(root)(/Users/Fj00/Ts/config.ru:3)
	at org.jruby.RubyBasicObject.instance_eval(org/jruby/RubyBasicObject.java:1533)
	at RUBY.initialize(/Users/Fj00/.rvm/gems/jruby-1.7.16/gems/rack-1.6.4/lib/rack/builder.rb:55)
	at RUBY.(root)(/Users/Fj00/Ts/config.ru:1)

